study.md

```diff
@@ -52,13 +53,13 @@ help manage view-state?
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+The code is easier to maintain
```

Comment:
This may be true but more importantly it allows our app's to not rely on the API for any updates to the page. We can dynamically render HTML to manage view states and navigate through the page for our user without waiting on data to come back from the server.